packages/block-editor/src/components/block-list/moving-animation.js

```diff
@@ -40,8 +40,8 @@ function useMovingAnimation( ref, isSelected, enableAnimation, triggerAnimationO
 		ref.current.style.transform = 'none';
 		const destination = ref.current.getBoundingClientRect();
 		const newTransform = {
-			x: previous ? previous.left - destination.left : 0,
-			y: previous ? previous.top - destination.top : 0,
+			x: previous ? previous.left - destination.left : -50,
```

Comment:
I think this negative margin may be causing a strange movement on the previous block when a new block is inserted:
![Jul-14-2019 14-34-42](https://user-images.githubusercontent.com/11271197/61184358-bff4f700-a644-11e9-962a-e19275bbc8b2.gif)

Reply:
That's strange. I would think `previous` is defined there. I'll look into it.